pom.xml

```diff
@@ -1037,23 +1103,55 @@
             </activation>
             <properties>
                 <envClassifier>mapr5.2.0</envClassifier>
-                <envHbase>hbase1.1.0</envHbase>
+                <envHbase>hbase1.1.1</envHbase>
                 <hadoop.version>2.7.0-mapr-1607</hadoop.version>
                 <hadoop.tools.version>1.0.3-mapr-5.2.0</hadoop.tools.version>
                 <hbase.version>1.1.1-mapr-1602-5.2.0</hbase.version>
                 <hive.version>0.13.0-mapr-1602</hive.version>
                 <zookeeper.version>3.4.5-mapr-1503</zookeeper.version>
                 <maprfs.version>5.2.0-mapr</maprfs.version>
-                <spark.version>2.2.0</spark.version>
+                <spark.major.version>spark2.1</spark.major.version>
+                <spark.version>2.1.0-mapr-1710</spark.version>
```

Comment:
is this supported on mapr 5.2.0? i struggled to find this build last October

Reply:
This is the spark provided with the 5.2 sandbox for MapR.  I usually validate the maven versions with what is actually being pushed by the vendors.  I did this as well for HDP if you wonder why a specific version is chosen.